test/Parse/structure_no_overflow.swift

```diff
@@ -35,3 +35,16 @@ let c = ({({({({ ({({({([ ({({({({ ({({({([  ({({({({ ({({({([ ({({({({ ({({({({
         })})})}) })})})}) ])})})}) })})})})  ])})})}) })})})}) ])})})}) })})})}) 
         })})})}) })})})}) ])})})}) })})})})  ])})})}) })})})}) ])})})}) })})})}) 
         })})})}) })})})}) ])})})}) })})})})  ])})})}) })})})}) ])})})}) })})})}) 
+
+// Test that we can have the allowed # of square brackets.
+// Note in bug SR-9220 this gave no error, but failed to parse in reasonable time.
+// CHECK-NOT: error
+let d = [[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[ [[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[
+        [[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[ [[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[
+        [[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[ [[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[
+        [[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[ [[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[
+         1
+         ]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]] ]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]
+        ]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]] ]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]
+        ]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]] ]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]
+        ]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]] ]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]
```

Comment:
I wonder how quickly does this parse now?

Reply:
let l = [[[[[[[[[[[[[[[ ]]]]]]]]]]]]]]]
        0.09 real         0.06 user         0.02 sys
let l = [[[[[[[[[[[[[[[[ ]]]]]]]]]]]]]]]]
        0.11 real         0.07 user         0.02 sys
let l = [[[[[[[[[[[[[[[[[ ]]]]]]]]]]]]]]]]]
        0.13 real         0.10 user         0.02 sys
let l = [[[[[[[[[[[[[[[[[[ ]]]]]]]]]]]]]]]]]]
        0.18 real         0.14 user         0.03 sys
let l = [[[[[[[[[[[[[[[[[[[ ]]]]]]]]]]]]]]]]]]]
        0.27 real         0.23 user         0.03 sys
let l = [[[[[[[[[[[[[[[[[[[[ ]]]]]]]]]]]]]]]]]]]]
        0.47 real         0.42 user         0.04 sys
let l = [[[[[[[[[[[[[[[[[[[[[ ]]]]]]]]]]]]]]]]]]]]]
        0.86 real         0.78 user         0.06 sys
let l = [[[[[[[[[[[[[[[[[[[[[[ ]]]]]]]]]]]]]]]]]]]]]]
        1.69 real         1.50 user         0.11 sys
let l = [[[[[[[[[[[[[[[[[[[[[[[ ]]]]]]]]]]]]]]]]]]]]]]]
        3.30 real         2.94 user         0.21 sys
let l = [[[[[[[[[[[[[[[[[[[[[[[[ ]]]]]]]]]]]]]]]]]]]]]]]]
        6.47 real         5.85 user         0.39 sys
let l = [[[[[[[[[[[[[[[[[[[[[[[[[ ]]]]]]]]]]]]]]]]]]]]]]]]]
       13.47 real        12.44 user         0.80 sys
let l = [[[[[[[[[[[[[[[[[[[[[[[[[[ ]]]]]]]]]]]]]]]]]]]]]]]]]]
       26.47 real        24.59 user         1.61 sys
let l = [[[[[[[[[[[[[[[[[[[[[[[[[[[ ]]]]]]]]]]]]]]]]]]]]]]]]]]]
       51.32 real        47.20 user         3.63 sys
let l = [[[[[[[[[[[[[[[[[[[[[[[[[[[[ ]]]]]]]]]]]]]]]]]]]]]]]]]]]]
      103.45 real        95.47 user         7.31 sys
let l = [[[[[[[[[[[[[[[[[[[[[[[[[[[[[ ]]]]]]]]]]]]]]]]]]]]]]]]]]]]]
      280.94 real       195.70 user        14.23 sys
let l = [[[[[[[[[[[[[[[[[[[[[[[[[[[[[[ ]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]
        463.93 real       386.87 user        27.48 sys

Reply:
Huh, 463.93 is still pretty long. Maybe it's the best we can do.